src/cpp/src/llm_pipeline.cpp

```diff
@@ -143,6 +156,7 @@ class StatefulLLMPipeline final : public LLMPipelineImplBase {
                     encoded_input = subtract_chat_tokenized_inputs(new_chat_tokens, prev_chat_tokens);
                 }
                 m_templated_chat_history = new_templated_chat_history;
+                // TODO: Forbid LoRA config change if we are in the chat mode, because it requires regenerating the history with LoRA applied
```

Comment:
is it planned to be fixed in next PRs before 2024.5 ?

Reply:
Yes. Added to TODO list in the PR description.